cli/doc/parser.rs

```diff
@@ -44,13 +44,15 @@ pub trait DocFileLoader {
 pub struct DocParser {
   pub ast_parser: AstParser,
   pub loader: Box<dyn DocFileLoader>,
+  pub private: bool,
```

Comment:
Great! I saw that it requires `--private` flag - which makes it a new feature. @SyrupThinker if that wouldn't be too much trouble it'd be great to split this PR into 2 smaller ones - that way we could land formatting refactor now and land `--private` option in `1.2.0`, but if that requires a lot of work then it's fine as is

Reply:
It'll be fine when it lands in `1.2.0`